secboot/secboot_dummy.go

```diff
@@ -27,3 +27,7 @@ import (
 func CheckKeySealingSupported() error {
 	return fmt.Errorf("build without secboot support")
 }
+
+func SealKey(key EncryptionKey, params *SealKeyParams) error {
+	return fmt.Errorf("build without secboot support")
```

Comment:
s/build/built/ and above too, or maybe something more generic for both errors eg. `secboot support not implemented`

Reply:
yep, otoh the other functions have the same problem it seems, so can be a follow up I suppose